cocoviewer.py

```diff
@@ -13,7 +12,7 @@
 import tkinter.ttk as ttk
 from tkinter import filedialog, messagebox
 from turtle import __forwardmethods
-
+import numpy as np
```

Comment:
```suggestion

import numpy as np
```
nit: add an empty line to separate imports